_posts/2019-11-27-update-sitemaps-during-heroku-deploy.markdown

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "Update sitemaps with dynamic content"
```

Comment:
maybe you can change the title to:

```suggestion
title:  "How to dynamically update your sitemap in Ruby"
```

Reply:
Much better! Thanks for the suggestion @lubc  :)